Fix ChoiceTile content padding

PavelHolec opened the pull request:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/35844477/211844639-ae557149-c069-48f2-9ec3-a4bb1bf00b53.png">

<img width="214" alt="image" src="https://user-images.githubusercontent.com/35844477/211844731-8d80260f-8673-42f9-81cb-136442b4a191.png">

sjavora commented:
Is the badge shift intended/wanted?

<img width="869" alt="Screenshot 2023-01-11 at 16 17 29" src="https://user-images.githubusercontent.com/12349477/211843769-20ffe6c0-e747-4560-b871-d9969a223253.png">

PavelHolec commented:
> Is the badge shift intended/wanted?
> 
> <img alt="Screenshot 2023-01-11 at 16 17 29" width="869" src="https://user-images.githubusercontent.com/12349477/211843769-20ffe6c0-e747-4560-b871-d9969a223253.png">

Should be fixed now. Updated the API to be more consistent with ListChoice, removing the (non-Orbit) badge and replaced by any custom header content.

sjavora commented on Sources/Orbit/Components/ChoiceTile.swift:
```diff
@@ -186,12 +199,23 @@ public struct ChoiceTile<Content: View>: View {
-        indicatorSize > 0
-            ? (indicatorSize + padding + .xSmall)
+    var indicatorTrailingPadding: CGFloat {
+//        indicatorSize > 0 && alignment == .default && headerContent is EmptyView
```
Commented code.

PavelHolec replied:
Thx, updated

sjavora commented:
Also conflicts.
